ci-operator/step-registry/vsphere/provision/bastionhost/vsphere-provision-bastionhost-commands.sh

```diff
@@ -21,6 +21,11 @@ if [[ ! -f "${bastion_ignition_file}" ]]; then
 fi
 bastion_ignition_base64=$(base64 -w0 <"${bastion_ignition_file}")
 
+if [[ -z "${vsphere_connected_portgroup}" ]]; then
+  echo "Not defined vsphere_connected_portgroup, bastion host will be provisioned in network defined as LEASED_RESOURCE..."
```

Comment:
Mentioning `LEASED_RESOURCE` may sound misleading as it is deprecated in vsphere.

Reply:
I think you can change `LEASED_RESOURCE ` to `vsphere_portgroup` in echo statement.

Reply:
@mhanss So in your case, the bastion host is needed, but the bastion is existing in the same network where the cluster is running?

Reply:
@jianlinliu QE also has job `vsphere-upi-disconnecting-network-f28` which needs bastion that resides on the same network as the cluster.

Reply:
@jianlinliu yes, and it would require a connected lease; otherwise, it won't be able to establish the mirror registry.

Reply:
@jrvaldes @jinyunma changes have been implemented, thanks.

Reply:
Good point, I think later maybe we should update the `vsphere_connected_portgroup` to `vsphere_bastion_portgroup`,  `VSPHERE_CONNECTED_LEASED_RESOURCE` to `VSPHERE_BASTION_LEASED_RESOURCE` for easy understanding.

Reply:
> it would require a connected lease; otherwise, it won't be able to establish the mirror registry

If that is the case, I think you should follow @jinyunma mentioned example in https://github.com/openshift/release/pull/44920#issuecomment-1782193764 to resolve it by add a explicit connected lease on the job config level.

Reply:
Yes, it's already present for my job execution except for the 4.12 version. I will create a separate pull request to add it. However, this condition is necessary for @saifshaikh48 scenarios because he's using a bastion host to set up the proxy server.

Reply:
Got it.